docs/Getting-Started.md

```diff
@@ -119,11 +123,12 @@ fastify.register(require('./our-db-connector'), {
 })
 fastify.register(require('./our-first-route'))
 
-fastify.listen(3000, function (err) {
+fastify.listen(3000, function (err, address) {
   if (err) {
     fastify.log.error(err)
     process.exit(1)
   }
+  console.log(`server listening on ${address}`)
```

Comment:
Use `fastify.log.info` instead of `console.log`.

Reply:
fixed